build.gradle.kts

```diff
@@ -91,6 +91,7 @@ kotlin {
             dependencies {
                 implementation(kotlin("test-common"))
                 implementation(kotlin("test-annotations-common"))
+                implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core-native:1.3.7")
```

Comment:
I was hoping we would be able to keep the coroutine dependencies test only, is there a reason we need to have an implementation dependency on it?

Reply:
this is test only, its added to `commonTest`